### PR TITLE
Feature | Change Help icon to a button that says User guide

### PR DESCRIPTION
**Pull request by pauline-n**
#664

### CODE REVIEW

File: frontend/src/components/Help/HelpNavItem.tsx [hunk 1]
```diff
@@ -18,7 +19,7 @@ const HelpNavItem: FunctionComponent = () => {
   return (
     <Dropdown as={Nav.Item} aria-labelledby="navbarDropdownHelp">
       <Dropdown.Toggle as={Nav.Link} id="help-nav-dropdown" data-cy="help">
-        <i className="material-icons">help</i>
+        <Button>User Guide</Button>
         <p className="d-lg-none d-md-block">Help</p>
```
Comment (ThatcherK) [inline frontend/src/components/Help/HelpNavItem.tsx]: This line is no longer needed, remove.. line 23

**Review by ThatcherK — CHANGES_REQUESTED** [review]

File: frontend/src/components/Help/HelpNavItem.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { FunctionComponent } from 'react';
 import { Dropdown, Nav } from 'react-bootstrap';
 import { Route, Switch } from 'react-router-dom';
+import { Button } from 'semantic-ui-react';
```
Comment (edwinmp) [inline frontend/src/components/Help/HelpNavItem.tsx]: Unless the component you need doesn't exist in Bootstrap, always use Bootstrap components, not Semantic UI.

**Review by edwinmp — CHANGES_REQUESTED** [review]

**Comment by pauline-n** [comment]
Screenshot 2022-05-23 at 11 47 00
@edwinmp @ThatcherK This is how it looks like

**Comment by edwinmp** [comment]
@pauline-n it's too big and it has a dropdown arrow beside it. Give it the class `btn-sm` or since it's bootstrap, give it a `size` prop set to the value `sm`

**Comment by pauline-n** [comment]
Screenshot 2022-05-23 at 12 39 20
Not sure if you had also meant, remove the dropdown beside it?

File: frontend/src/components/Help/HelpNavItem.tsx [hunk 3]
```diff
@@ -18,8 +18,9 @@ const HelpNavItem: FunctionComponent = () => {
   return (
     <Dropdown as={Nav.Item} aria-labelledby="navbarDropdownHelp">
       <Dropdown.Toggle as={Nav.Link} id="help-nav-dropdown" data-cy="help">
-        <i className="material-icons">help</i>
-        <p className="d-lg-none d-md-block">Help</p>
+        <Button variant="btn-secondary" className="btn-secondary" size="sm">
```
Comment (edwinmp) [inline frontend/src/components/Help/HelpNavItem.tsx]: Incorrect variant ... I'm pretty sure this should result in an error.
```suggestion
        <Button variant="secondary" size="sm">
```

**Review by edwinmp — CHANGES_REQUESTED** [review]
Add an updated photo.

**Comment by edwinmp** [comment]
> Not sure if you had also meant, remove the dropdown beside it?

Ofc ... can't see any other dropdown arrow. This button is not a dropdown so it shouldn't have a little arrow that should only be on dropdowns.

**Comment by pauline-n** [comment]
Screenshot 2022-05-24 at 10 57 59
updated photo

**Comment by pauline-n** [comment]
Screenshot 2022-05-24 at 10 58 17
How it appears when you click on the button user guide

**Review by edwinmp — APPROVED** [review]

**Pull request merged**